assets/css/v2/lcd_common_styles/subway_status.scss

```diff
@@ -134,9 +137,18 @@
     width: fit-content;
     max-width: 830px;
     margin-left: 24px;
+
+    &--2-branches {
+      max-width: 671px;
+    }
+
+    &--3-branches {
+      max-width: 605px;
+    }
```

Comment:
Does this address a different problem than "[GL multi-branch station closures omit stops](https://www.notion.so/mbta-downtown-crossing/GL-multi-branch-station-closures-omit-stops-867ac83ee89b4feaa9c1d80386e3d1e6?pvs=4)"? Wondering what issue this fixes.

I think I found that the  `max-width` property didn't really matter, because the element has `width: fit-content` and either gets shrunk to fit within the row, or overflows and gets clipped at the right edge of the row. So its width can be whatever in either case.

It should still wrap while `useTextResizer` is doing its thing—that is, while `isDone` is false—since [the sizer element sets a width](https://github.com/mbta/screens/blob/56a6c5e612f3f65eb266cef4fcf133bdea629c63/assets/css/v2/lcd_common_styles/subway_status.scss#L113-L116). Or at least it did the last time I messed with the resizing stuff

Reply:
I found that `max-width` was working, but in most cases the string never gets close to hitting it. As you add more branch pills, that available space goes down. So say you have a station closure at Saint Mary's Street and Sutherland Road: 

![Screenshot 2023-06-30 at 12 08 50 PM](https://github.com/mbta/screens/assets/10713153/97baad23-9b14-4fc1-b80f-ac844454be20)

The text is still less than the max-width so it renders assuming everything fits. This causes the resizer to think it is ok rendering the station names as-is. If you reduce the max-width to what I have above, the resizer knows that there isn't enough space for the full station names so it tries the next method:

![Screenshot 2023-06-30 at 12 11 09 PM](https://github.com/mbta/screens/assets/10713153/b06056b0-66af-4810-863b-842d4d0a122a)

Does that make sense?

Reply:
Oh weird, yeah that's not what I had expected. I thought I had it wrapping because it exceeded the widget of the containing `sizer` element, and had the appropriate properties set up there. I must not have checked properly for this case. Good catch.